src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const marriedMessage = isMarried ? (
+    <p className="Person__partner">
+      {`${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`}
+    </p>
+  ) : (
+    <p className="Person__partner">I am not married</p>
+  );
```

Comment:
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```